images/stolon/Dockerfile

```diff
@@ -1,22 +1,18 @@
-ARG PGVERSION=9.6
+ARG PGVERSION=12
 # base build image
-FROM quay.io/gravitational/debian-venti:go1.13.8-buster AS builder
-ARG STOLON_REVISION=6353dbc5542d7243bd2bd5256f0a945fdc1f5c23
+FROM quay.io/gravitational/debian-venti:go1.16.3-buster AS builder
+ARG STOLON_REVISION=v0.16.0
```

Comment:
Also, would be nice to move all configuration outside of the dockerfiles - i.e. be able to override versions from a makefile - not sure this is the case now.

Reply:
ARG could be used outside of docker files with the `--arg` parameter. So, yes, I will move the values into Makefile.